src/utils/vc/credentials.js

```diff
@@ -27,7 +30,39 @@ function getId(obj) {
   if (typeof obj === 'string') {
     return obj;
   }
-  return obj.id;// || 'did:dock:' + String(obj);
+  return obj.id;
+}
+
+export function formatToJWTPayload(keyDoc, cred) {
+  const kid = keyDoc.id;
+  const credentialIssuer = cred.issuer;
+  const subject = cred.credentialSubject.id;
+  const validFrom = cred.validFrom || cred.issuanceDate;
+  const { issuanceDate, expirationDate } = cred;
+
+  // Reference: https://www.w3.org/TR/vc-data-model/#jwt-encoding
+  const vcJwtPayload = {
+    jti: cred.id,
+    sub: subject || '',
+    iss: credentialIssuer.id || credentialIssuer,
+    iat: Math.floor(Date.parse(issuanceDate) / 1000),
+    vc: cred,
+  };
+
+  if (validFrom) {
+    vcJwtPayload.nbf = Math.floor(Date.parse(validFrom) / 1000);
+  }
+
+  if (expirationDate) {
+    vcJwtPayload.exp = Math.floor(Date.parse(expirationDate) / 1000);
```

Comment:
You can abstract this to a function `Math.floor(Date.parse(expirationDate) / 1000)` and call that for this and above.